src/components/DecksPickBreakdown.js

```diff
@@ -125,7 +136,17 @@ const DecksPickBreakdown = ({ draft, seatIndex, deck, defaultIndex }) => {
 
 DecksPickBreakdown.propTypes = {
   draft: PropTypes.shape({
-    initial_state: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.array)).isRequired,
+    initial_state: PropTypes.arrayOf(
+      PropTypes.arrayOf(
+        PropTypes.shape({
+          cards: PropTypes.arrayOf(PropTypes.number).isRequired,
```

Comment:
Reading this I'm wondering if its more or less confusing to name this `cards` something that makes it more obvious it is a number, since we have another cards just below that is not. Two things named the same with different data makes me worried.